### PR TITLE
Test/code review

### PR DESCRIPTION
**Pull request by mbadres-recom**

### CODE REVIEW

File: HelloWorld/HelloWorld/Calculator/Engine.cs [hunk 1]
```diff
@@ -24,6 +24,12 @@ public double Divide(double a, double b)
             if (b == 0) return Math.PI;
             return a / b;
         }
+        
+        public double Pow(double a, double b)
+        {
+            if (b == 0) return Math.PI;
+            return a / b;
+        }
```
Comment (KlausSeidl) [inline HelloWorld/HelloWorld/Calculator/Engine.cs]: I think, that not Pow. It returns Pi or fraction

File: HelloWorld/HelloWorld/ViewModel/MainViewModel.cs [hunk 2]
```diff
@@ -39,14 +39,14 @@ public ICommand GreetCommand
...
-            ////    // Code runs "for real"
-            ////}
+            if (IsInDesignMode)
+            {
+                // Code runs in Blend --> create design time data.
+            }
+            else
+            {
+                // Code runs "for real"
+            }
```
Comment (KlausSeidl) [inline HelloWorld/HelloWorld/ViewModel/MainViewModel.cs:49]: Why not delete at all? Code does nothing

File: HelloWorld/HelloWorld/ViewModel/ViewModelLocator.cs [hunk 3]
```diff
@@ -30,6 +31,7 @@ public class ViewModelLocator
         public ViewModelLocator()
         {
             ServiceLocator.SetLocatorProvider(() => SimpleIoc.Default);
+            ServiceLocator.SetLocatorProvider(() => new Foo());
```
Comment (KlausSeidl) [inline HelloWorld/HelloWorld/ViewModel/ViewModelLocator.cs]: Does not compile. Did you forget to add Foo to repository

**Review by KlausSeidl — CHANGES_REQUESTED** [review]
Not bad, but can be improved

File: HelloWorld/HelloWorld/ViewModel/MainViewModel.cs (hunk 2 above)
Comment (mbadres-recom) [reply HelloWorld/HelloWorld/ViewModel/MainViewModel.cs:49]: We will need that, because ...

Comment (KlausSeidl) [reply HelloWorld/HelloWorld/ViewModel/MainViewModel.cs:49]: Ah, I understand. That's fine

**Review by KlausSeidl — APPROVED** [review]
Well done

**Pull request closed without merging**